AdminCore.WebApp/Startup.cs

```diff
@@ -58,6 +57,9 @@ public void ConfigureServices(IServiceCollection services)
         });
       });
 
+      services.AddSingleton<IMailServerConfiguration>(
```

Comment:
Hmm  I think this should take the IConfiguration instead in the constructor. Shouldn't need to specify constructor args here.